superset-frontend/src/assets/stylesheets/superset.less

```diff
@@ -518,9 +518,14 @@ tr.reactable-column-header th.reactable-header-sortable {
   padding-right: 17px;
 }
 
+.metric-edit-popover-label {
+  display: inline-block;
+  padding: 2px 0;
+}
+
 .metric-edit-popover-label-input {
   border-radius: @border-radius-large;
-  height: 30px;
+  height: 26px;
```

Comment:
Would it be possible to move both these classes to Emotion styling in the component? We're trying to whittle away at these LESS files so that we can use variables from the Superset theme everywhere (e.g. using the `gridUnit` and `borderRadius` keys from the theme)